diff --git a/rpcs3/rpcs3qt/config_database.cpp b/rpcs3/rpcs3qt/config_database.cpp
index 6f8f4e8..87bf464 100644
--- a/rpcs3/rpcs3qt/config_database.cpp
+++ b/rpcs3/rpcs3qt/config_database.cpp
@@ -1,5 +1,9 @@
 #include "stdafx.h"
 #include "config_database.h"
+#include <QJsonDocument>
+#include <QJsonParseError>
+#include <QJsonObject>
+#include <QJsonArray>
 #include "gui_settings.h"
 #include "downloader.h"
 #include "Emu/system_config.h"
diff --git a/rpcs3/rpcs3qt/game_list_actions.cpp b/rpcs3/rpcs3qt/game_list_actions.cpp
index c8402ae..90c760a 100644
--- a/rpcs3/rpcs3qt/game_list_actions.cpp
+++ b/rpcs3/rpcs3qt/game_list_actions.cpp
@@ -550,7 +550,7 @@ void game_list_actions::ShowGameIntegrityDialog(content_file_type file_type, con
 				0;
 
 			pdlg->setValue(progress);
-			pdlg->setLabelText(tr("Calculating hash: %0").arg(m_game_validator->get_name()));
+			pdlg->setLabelText(tr("Calculating hash: %0").arg(QString::fromStdString(m_game_validator->get_name())));
 		}
 		else
 		{
diff --git a/rpcs3/rpcs3qt/game_list_base.h b/rpcs3/rpcs3qt/game_list_base.h
index 6997594..4d07742 100644
--- a/rpcs3/rpcs3qt/game_list_base.h
+++ b/rpcs3/rpcs3qt/game_list_base.h
@@ -4,6 +4,7 @@
 
 #include <QIcon>
 #include <QWidget>
+#include <set>
 
 class game_list_base
 {
